ignite/contrib/handlers/time_profilers.py

```diff
@@ -0,0 +1,239 @@
+from collections import OrderedDict
+
+import torch
+
+from ignite.engine import Engine, Events
+from ignite.handlers import Timer
+
+
+class BasicTimeProfiler(object):
+
```

Comment:
Here we need to add docstring. For example,https://github.com/pytorch/ignite/blob/4253bfbf40011288ed952e73344e8d470e164b9f/ignite/handlers/early_stopping.py#L7